build.gradle

```diff
@@ -44,6 +44,7 @@ configurations.all {
         force group: 'com.google.errorprone', name: 'error_prone_annotations', version: '2.18.0'
         force group: 'org.checkerframework', name: 'checker-qual', version: '3.33.0'
         force group: 'com.google.j2objc', name: 'j2objc-annotations', version: '2.8'
+        force group: 'com.github.ben-manes.caffeine', name:'caffeine', version: '2.9.3'
```

Comment:
why do we need to add this?

Reply:
Latest versions of script-security', '1369.v9b_98a_4e95b_2d requires caffeine version 3.1.8 which is not compatible with our java Version
check the failure [here](https://github.com/opensearch-project/opensearch-build/actions/runs/12380186968/job/34556060483)

Reply:
Why are we still pointing to java8? Isnt 11 or 17 is the one we use now?

Reply:
Or is there a version for us to upgrade to, that resolves the cve but not require a lower version of caffeine?